src/Interpreters/Streaming/GetSampleBlockContext.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <Core/Streaming/DataStreamSemantic.h>
+
+namespace DB
+{
+namespace Streaming
+{
+struct GetSampleBlockContext
```

Comment:
No need to add this struct and related function. If 'is_streaming_output' put into DataStreamSemanticEx